pkg/controller/import-controller.go

```diff
@@ -187,8 +197,8 @@ func (ic *ImportController) syncPvc(key string) error {
 	if pvc == nil {
 		return nil
 	}
-	//check if AnnoEndPoint annotation exists
-	if !checkPVC(pvc, AnnEndpoint) {
+	//check if AnnEndPoint or AnnSource annotation exists
+	if !checkPVC(pvc, AnnEndpoint) && !checkPVC(pvc, AnnSource) {
```

Comment:
If the EndPoint anno exists and Source anno does not, we still processing the PVC. I think the source annotation should be 'must' in order to keep processing the PVC. Otherwise it's very confusing. The source annotation is not used only for Enpoint case of course. Look also at my other comments related to this.

Reply:
See the other answers, its to maintain backwards compatibility. Remember fairly soon we will have proper data sources on PVCs themselves and we won't maintain the backwards compatibility at that point.